src/RealtimeClient.ts

```diff
@@ -317,6 +321,77 @@ export default class RealtimeClient {
     })
   }
 
+  /**
+   * Creates a private channel to be used with the provided name.
+   *
+   * @param name Channel name to create. If null a random channel name will be generated
+   */
+  createPrivateChannel(name: string | null): Promise<string> {
+    let channelName = name || Math.random().toString(36).substring(2, 15)
```

Comment:
@filipecabaco we should go with a solution that's crypto secure.

Reply:
agreed! I'm a bit afraid of using libs for this and I'm unsure how to get the proper crypto lib loaded.

we could use https://developer.mozilla.org/en-US/docs/Web/API/Crypto but I saw somewhere this actually differs from runtime to runtime.

wdyt?

Reply:
@filipecabaco I think we should require channel name just to make it easier for ourselves. Can always do this later when necessary.

Reply:
removed the random name generation so we can discuss it at a later stage